libs/@hashintel/design-system/src/autocomplete-dropdown.tsx

```diff
@@ -37,50 +38,53 @@ export const AutocompleteDropdown = ({
       />
       <Paper
         {...props}
-        sx={(theme) => ({
-          p: 1,
-          border: 1,
-          boxSizing: "border-box",
-          borderColor: theme.palette.gray[30],
-          boxShadow: "none",
-          [`${popperPlacementSelectors.top} &`]: {
-            borderBottom: 0,
-            borderBottomLeftRadius: 0,
-            borderBottomRightRadius: 0,
-          },
-          [`${popperPlacementSelectors.topStart} &`]: {
-            borderBottom: 0,
-            borderBottomLeftRadius: 0,
-          },
-          [`${popperPlacementSelectors.bottom} &`]: {
-            borderTop: 0,
-            borderTopLeftRadius: 0,
-            borderTopRightRadius: 0,
-          },
-          [`${popperPlacementSelectors.bottomStart} &`]: {
-            borderTop: 0,
-            borderTopLeftRadius: 0,
-          },
-
-          [`.${autocompleteClasses.listbox}`]: { p: 0 },
-          [`.${autocompleteClasses.noOptions}`]: { display: "none" },
-          [`.${autocompleteClasses.option}`]: {
-            borderRadius: 1,
-            my: 0.25,
-
-            [`&[aria-selected="true"]`]: {
-              backgroundColor: `${theme.palette.blue[20]} !important`,
+        sx={[
+          (theme) => ({
+            p: 1,
+            border: 1,
+            boxSizing: "border-box",
+            borderColor: theme.palette.gray[30],
+            boxShadow: "none",
+            [`${popperPlacementSelectors.top} &`]: {
```

Comment:
For my understanding, what does the `&` do here?

Reply:
I wasn't sure myself, as this was taken from the AutocompleteDropdown in the design-system. But [here](https://sass-lang.com/documentation/style-rules/parent-selector)'s what I found